src/lib/content/types.ts

```diff
@@ -1,150 +1,215 @@
-export interface Location {
-  uuid?: string,
+type Days = 'MON' | 'TUE' | 'WED' | 'THU' | 'FRI' | 'SAT' | 'SUN'
+
+export interface Schema {
+  type: string,
+  custom_fields?: Array<{
+    name: string,
+    value: string,
+  }>
+}
+
+export interface Phone {
+  phoneNumber: string,
+  label: string,
+}
+
+export interface Email {
+  emailAddress: string,
+  label: string,
+}
+
+export interface SocialAccounts {
+  facebook?: string,
+  twitter?: string,
+  yelp?: string,
+  foursquare?: string,
+  instagram?: string,
+  youtube?: string,
+  linkedin?: string,
+  pinterest?: string,
+  vimeo?: string,
+  rss?: string,
+  reddit?: string,
+  tripadvisor?: string,
+  snapchat?: string,
+}
+
+export interface Address {
+  streetAddress?: string,
+  postalCode?: string,
+  region?: string,
+  city?: string,
+  country?: string,
+}
+
+export interface Geo {
+  longitude?: string | null,
+  latitude?: string | null,
+}
+
+export interface BusinessHours {
+  days: Days[],
+  open: string,
+  close: string,
+}
+
+export interface SiteTexts {
+  overview: string,
+  services: string,
+  about_us: string,
+  custom: Array<{
+    label: string,
+    value: string,
+  }>,
+}
+
+export interface CreateLocationResponse {
+  uuid: string,
+  phones: Phone[] | null,
+  emails: Email[] | null,
+  social_accounts: {
+    socialAccounts: SocialAccounts
+  } | null,
+  address: Address | null,
+  geo: Geo | null
+  label: string | null,
+  logo_url: string | null,
+  business_hours: BusinessHours[] | null,
+}
+
+export interface GetLocationResponse {
+  uuid: string,
+  phones: Phone[],
+  emails: Email[],
+  social_accounts: {
+    socialAccounts: SocialAccounts
+  }
+  address_geolocation: string,
+  geo: {
+    longitude: string | null,
+    latitude: string | null
+  },
+  logo_url: string | null,
+  business_hours: BusinessHours[],
+  label: string | null,
+  address: Address
+}
+
+export interface MainLocationResponse extends GetLocationResponse {
+  schema?: Schema,
+  address_geolocation: string,
```

Comment:
is this redundant since address_geolocation is a parameter in GetLocationResponse?